src/nni_manager/common/trainingService.ts

```diff
@@ -22,7 +22,7 @@
 /**
  * define TrialJobStatus
  */
-type TrialJobStatus = 'UNKNOWN' | 'WAITING' | 'RUNNING' | 'SUCCEEDED' | 'FAILED' | 'USER_CANCELED' | 'SYS_CANCELED';
+type TrialJobStatus = 'UNKNOWN' | 'WAITING' | 'RUNNING' | 'SUCCEEDED' | 'FAILED' | 'USER_CANCELED' | 'SYS_CANCELED' | 'EARLY_STOPPED';
```

Comment:
Since this change will impact all the training services(local/remote/pai/kubeflow), I suggest to do regression tests on all training services, either by yourself or someone's help

Reply:
agree, let's arrange this in next scrum